src/data/pokemon/pokedex_text.h

```diff
@@ -5349,4 +5349,44 @@ const u8 gCalyrexPokedexText[] = _(
     "providing healing and blessings. It\n"
     "reigned over the Galar region in times\n"
     "of yore.");
+
+const u8 gWyrdeerPokedexText[] = _(
+    "The black orbs shine with an uncanny light\n"
+    "when it is erecting invisible barriers.\n"
+    "The fur shed from its beard retains heat\n"
+    "and useful for winter clothing.");
+
+const u8 gKleavorPokedexText[] = _(
+    "A violent creature that fells trees with\n"
+    "its crude axes and shields itself with hard\n"
+    "stone. If one should encounter this Pokémon\n"
+    "in the wild, one's only recourse is to flee.");
+
+const u8 gUrsalunaPokedexText[] = _(
+    "Swampy terrain gives Ursaluna its burly\n"
+    "physique and newfound capacity to\n"
+    "manipulate peat at will.");
+
+const u8 gBasculegionPokedexText[] = _(
+    "It clads itself in the souls of comrades\n"
+    "that perishes before their goals of\n"
+    "journeying upstream.");
+
+const u8 gSneaslerPokedexText[] = _(
+    "Because of its wicked poison and daunting\n"
+    "physical prowess, no other species could\n"
+    "hope to best it on frozen highlands. It\n"
+    "prefers solitude and doesn't form packs.");
+
+const u8 gOverqwilPokedexText[] = _(
+    "Its lancelike spikes and savage\n"
+    "temperament have earned it the\n"
+    "nickname Sea Fiend.\n"
+    "It surps up poison to nourish itself.");
```

Comment:
```suggestion
    "nickname 'sea fiend'.\n"
    "It slurps up poison to nourish itself.");
```

Reply:
Should be "“Sea Fiend.”" not 'sea fiend' like in other existing descriptions such as Landorus' imo.

Reply:
Forgot how to do quotes, fixed. But I chose lower case, as that's how it is in its entry in PLA. 
![IMG_20220827_075025_783](https://user-images.githubusercontent.com/2904965/187028957-f734b899-66da-4b84-a27c-e8b6e2e08c88.jpg)

Reply:
LA didn't have to worry about keeping consistency being kind of detached from the mainline games and championing entirely new descriptions, so I stand by "Sea Fiend" with smart quotes for consistency with the other dex entries :eyes:

Reply:
Fair. Updated.